code/modules/food_and_drinks/food/snacks_cake.dm

```diff
@@ -308,16 +308,16 @@
 /obj/item/reagent_containers/food/snacks/store/cake/bscc // blackbarry strawberries chocolate cake
 	name = "blackberry and strawberry chocolate cake"
 	desc = "A chocolate cake, filled with assortment of blackberries and strawberries!"
-	icon_state = "blackbarry_strawberries_cake_coco_cake"
+	icon_state = "blackbarry_strawberries_cake_cocoa_cake"
```

Comment:
You rename icon states without editing DMIs.

Reply:
Noted, changed. Also blackbarry gonna make me act up